### DISCUSSION TITLE
How to remove opned history

### DISCUSSION THREAD

**Discussion by kikushyu** [sumatrapdfreader/sumatrapdf#3114] [General]
Hi! I had canceled to remember opened files from options.
But when I open a pdf file and right-click on it, opened files history appeared.
How to remove hostory completely.
Thanks! 
[pic1](https://imgur.com/4aIFNha)
[pic2](https://imgur.com/fq8m4GC)

**Comment by GitHubRulesOK** [comment]
Thats **Windows** Remember recent files outside NOT inside SumatraPDF
you need a Window Cleaner
